go/felix/iptables/table_test.go

```diff
@@ -436,19 +472,23 @@ var _ = Describe("Table with a dirty dataplane", func() {
 			})
 		})
 		checkFinalState := func() {
-			Expect(dataplane.Chains).To(Equal(map[string][]string{
+			expChains := map[string][]string{
 				"FORWARD": {
-					"-m comment --comment \"cali:hecdSCslEjdBPBPo\" --jump DROP",
-					"-m comment --comment \"cali:plvr29-ZiKUwbzDV\" --jump ACCEPT",
 					"--jump RETURN",
 					"--jump ACCEPT",
 					"--jump foo-bar",
+					"-m comment --comment \"cali:hecdSCslEjdBPBPo\" --jump DROP",
+					"-m comment --comment \"cali:plvr29-ZiKUwbzDV\" --jump ACCEPT",
```

Comment:
Is there a reason for setting the FORWARD rules here, given that they get overwritten immediately below?  I think it would be clearer to omit them here.